lib/shopify-cli/messages/messages.rb

```diff
@@ -6,13 +6,14 @@ module Messages
       core: {
         connect: {
           help: <<~HELP,
-          Connect a Shopify App CLI project. Restores the ENV file.
+          Connect a project to use Shopify App CLI functionality. Creates or updates .env file, creates .shopify-cli.yml file.
```

Comment:
Suggested wording to clarify what this command actually does (cc @gfscott )

```suggestion
          Connect (or re-connect) an existing project to a Shopify partner organization and/or a store. Creates or updates the {{green:.env}} file, and creates the {{green:.shopify-cli.yml}} file.
```